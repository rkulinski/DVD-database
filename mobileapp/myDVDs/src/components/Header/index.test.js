import React from 'react';
import { Header } from './';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
