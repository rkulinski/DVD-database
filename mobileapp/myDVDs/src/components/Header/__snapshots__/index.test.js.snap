// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "height": 52.5,
        "width": "100%",
      },
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontSize": 37.5,
          "fontWeight": "bold",
          "textAlign": "center",
        },
      ]
    }
  />
</View>
`;
